### ISSUE TITLE
Sound is broken on Windows

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#1371]
I guess it's the SDL2 update, seems weird in Wine.

**Comment by def-** [comment]
Doesn't look t hat bad when initializing either:
```
0045:fixme:msctf:ThreadMgrSource_AdviseSink (0xb67d0) Unhandled Sink: {ea1ea136-19df-11d7-a6d2-00065b84435c}
0045:fixme:imm:NotifyIME NI_CLOSECANDIDATE
0045:fixme:msctf:ThreadMgrSource_AdviseSink (0xb67d0) Unhandled Sink: {ea1ea136-19df-11d7-a6d2-00065b84435c}
004b:fixme:avrt:AvSetMmThreadCharacteristicsW (L"Pro Audio",0x214fcdc): stub
[2018-11-01 14:19:28][client/sound]: sound init successful
```

**Comment by xse** [comment]
Confirmed on windows.

**Comment by def-** [comment]
Anyone who can test if just dropping in the old SDL2 version fixes it? Or SDL 2.0.9 that was just released?

**Comment by Ryozuki** [comment]
Dropping SDL 2.0.5 fixed it for me. I couldn't try with 2.0.6 2.0.7 because i can't find where to download them.
